package.json

```diff
@@ -23,7 +23,7 @@
     "start": "node scripts/start.js",
     "start-mainnet": "REACT_APP_NETWORK=mainnet yarn start",
     "test": "NODE_ENV=test && node scripts/test.js --env=jsdom",
-    "format": "prettier-eslint \"src/**/*.js\" --write"
+    "format": "prettier-eslint '$PWD/src/**/*.{js,jsx}' --write"
```

Comment:
does this work for windows users?

Reply:
I doubt it'll work for Windows users without some extra configuration by the system's user, but the way it was before seemed to work for nobody.

I was getting thousands of errors like this:

```There was an error formatting "src/routes/safe/store/models/confirmation.js": 
    AssertionError [ERR_ASSERTION]: 'basePath' should be an absolute path.```

Also it seemed to be missing `.jsx` files as well, I guess.

Please check if it worked for you the way it was before. `prettier-eslint` doesn't seem be doing anything to formatted or unformatted files, BTW.

Reply:
This should explain the situation better https://github.com/prettier/prettier-eslint-cli/issues/208#issuecomment-561093777

Reply:
Thanks to @fernandomg who found a typo, the ` "prettier-eslint $PWD'/src/**/*.{js,jsx}' --write"` command works now.

According to another developer in the team this app doesn't work or it's pretty difficult to make it work in Windows, so I'm guessing this shouldn't be a problem?

Reply:
I use prettier-vscode, never ran this command :D

Reply:
maybe https://www.npmjs.com/package/cross-env can help
not critical at all though

Reply:
Yeah, I had `prettier-vscode` running, but it was acting wonky along with ESLINT. Couldn't find a balance between the two.

I'll enable it again and see what it does, it should do the same than the formatting script.